romancal/flatfield/tests/test_flatfield.py

```diff
@@ -20,7 +20,7 @@ def test_flatfield_step_interface(instrument, exptype):
 
     shape = (20, 20)
 
-    wfi_image = maker_utils.mk_level2_image(shape=shape)
+    wfi_image = stnode.WfiImage.fake_data(shape=shape)
```

Comment:
This test could be restructured (in a follow-up PR) to no longer make a node but instead make an ImageModel directly.